draft-ietf-scitt-architecture.md

```diff
@@ -202,6 +157,19 @@ Many of the data elements processed by the core components are based on the CBOR
 
 The building blocks defined in SCITT are intended to support applications in any supply chain that produces or relies upon digital artifacts, from the build and supply of software and IoT devices to advanced manufacturing and food supply.
 
+## Relation to Certificate Transparency
+
+SCITT is a generalization of Certificate Transparency {{-CT}}, which can be interpreted as a transparency architecture for the supply chain of X.509 certificates.
+Considering CT in terms of SCITT:
+
+- CAs (Issuers) sign X.509 TBSCertificates (Artifacts) to produce X.509 certificates (Signed Statements)
```

Comment:
Its my understanding that CT includes modified certificates in the log.

So I might expect the following alignment:

```
TBSCertificates -> Certificate -> Deterministically Broken Cert -> Log Entry
Statement --> Signed Statement -> Log Entry
```

I am not a CT expert, but I would focus on what goes on the log and make it clear that Logs are about "what goes on them".

Reply:
The tree leaves store the TBSCertificate, among other things:
https://datatracker.ietf.org/doc/html/rfc9162#name-merkle-tree-leaves

Reply:
@OR13, if you're satisfied, can you mark this resolved?